RealEstate/RealEstate/Pages/Index.cshtml

```diff
@@ -73,14 +71,18 @@
                 foreach (LandData.Lands land in lands)
                 {
                     landRecords = landRecords + 1;
+                   
```

Comment:
//Change 1 -If Zip code is 0 remove the entire row

Reply:
Is ZipCode the only required field?  What if that is empty, and others are populated?